From: Nick Mathewson <nickm@torproject.org>
Date: Thu, 20 Mar 2014 17:52:50 +0000 (-0400)
Subject: Fix unittest compilation with --disable-curve25519
X-Git-Tag: tor-0.2.5.3-alpha~3^2
X-Git-Url: https://gitweb.torproject.org?hp=9077118ee27e275d495b89a810ee28882c74cf62

Fix unittest compilation with --disable-curve25519

This is a fix for 9700, which we already fixed in 0.2.5.x, but which
got left in 0.2.4.x.

This is a partial backport of a0a855d586d99540277014ccd3
---

diff --git a/changes/bug9700 b/changes/bug9700
new file mode 100644
index 0000000..f59f54c
--- /dev/null
+++ b/changes/bug9700
@@ -0,0 +1,3 @@
+  o Minor bugfixes (compilation):
+    - Fix a compilation error when compiling with --disable-cuve25519.
+      Fixes bug 9700; bugfix on 0.2.4.17-rc.
diff --git a/src/test/test.c b/src/test/test.c
index 4ec8792..c2911d8 100644
--- a/src/test/test.c
+++ b/src/test/test.c
@@ -56,12 +56,12 @@ double fabs(double x);
 #include "memarea.h"
 #include "onion.h"
 #include "onion_tap.h"
+#include "onion_ntor.h"
 #include "policies.h"
 #include "rephist.h"
 #include "routerparse.h"
 #ifdef CURVE25519_ENABLED
 #include "crypto_curve25519.h"
-#include "onion_ntor.h"
 #endif
 
 #ifdef USE_DMALLOC
